src/openrct2/world/TileInspector.cpp

```diff
@@ -343,7 +343,7 @@ namespace OpenRCT2::TileInspector
                     return GameActions::Result(GameActions::Status::Unknown, STR_TOO_MANY_BANNERS_IN_GAME, STR_NONE);
                 }
                 // Copy the banners style
-                *newBanner = *GetBanner(bannerIndex);
+                newBanner = GetBanner(bannerIndex);
```

Comment:
This isn't what you want to do. You would need to verify that the bannerIndex is creating a valid banner. I assume that GetBanner(bannerIndex) is returning nullptr. You just need to check for that and if it happens then return a failure. If you want to be extra helpful you would do this before the `if (isExecuting)`

Reply:
I understand now. Thanks for the feedback, I will adjust accordingly. Yeah GetBanner is returning a nullptr so will check for that.